developer_manual/app_publishing_maintenance/app_upgrade_guide/upgrade_to_29.rst

```diff
@@ -0,0 +1,83 @@
+=======================
+Upgrade to Nextcloud 28
+=======================
+
+General
+-------
+
+info.xml
+^^^^^^^^
+
+Make sure your ``appinfo/info.xml`` allows for Nextcloud 29.
+
+.. code-block:: xml
+
+    <dependencies>
+        <nextcloud min-version="27" max-version="29" />
+    </dependencies>
+
+Front-end changes
+-----------------
+
+Added APIs
+^^^^^^^^^^
+
+* tbd
+
+Changed APIs
+^^^^^^^^^^^^
+
+* tbd
+
+Deprecated APIs
+^^^^^^^^^^^^^^^
+
+* tbd
+
+Removed APIs
+^^^^^^^^^^^^
+
+* tbd
+
+Back-end changes
+----------------
+
+Added APIs
+^^^^^^^^^^
+
+* tbd
+
+Changed APIs
+^^^^^^^^^^^^
+
+* tbd
+
+Deprecated APIs
+^^^^^^^^^^^^^^^
+
+* tbd
+
+Removed APIs
+^^^^^^^^^^^^
+
+* tbd
+
+Added events
+^^^^^^^^^^^^
+
+* tbd
+
+Deprecated events
+^^^^^^^^^^^^^^^^^
+
+* tbd
+
+Removed events
+^^^^^^^^^^^^^^
+
+* tbd
+
+Changed behavior
+^^^^^^^^^^^^^^^^
+
+The dashboard no longer loads the sidebar or Viewer scripts, if your dashboard widget relys on this it should emit the required events itself.
```

Comment:
```suggestion
The dashboard no longer loads the sidebar or Viewer scripts, if your dashboard widget relies on this it should emit the required events itself.
```

nit: do we have detailed documentation to link to for *emit the event*?

Reply:
Sadly we have no viewer docs currently here but only on the viewer repo.